mover.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"fmt"
+	"github.com/k0kubun/go-ansi"
+	"github.com/schollz/progressbar/v3"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+func modifyFileName(targetDir string, imageInfo ImageInfo) string {
```

Comment:
Are you sure this code will compile? I can't see a `return` statement other than the one inside the `for`.